project-base/storefront/components/Pages/Stores/StoresContent.tsx

```diff
@@ -21,40 +19,22 @@ type StoresContentProps = {
 export const StoresContent: FC<StoresContentProps> = ({ stores }) => {
     const { t } = useTranslation();
     const { defaultLocale } = useDomainConfig();
-    const [activeStoreIndex, setActiveStoreIndex] = useState<number>();
+    const [activeStoreId, setActiveStoreId] = useState<string>();
```

Comment:
Just for clarity, since we are calling it `identifier` when working with stores, maybe I would keep that naming. Just a suggestion. No strong opinion here.

Reply:
Ok, that makes sense.